requirements.txt

```diff
@@ -5,3 +5,4 @@ scipy==0.17.1
 matplotlib==1.5.3
 pandas==0.19.2
 scikit-learn==0.18
+jsonschema
```

Comment:
We should add the specific version we're using just so that if they change some api in the future, it doesn't break our code.